Framework/DataObjects/inc/MantidDataObjects/MDEventWorkspace.tcc

```diff
@@ -1011,6 +1011,13 @@ TMDE(void MDEventWorkspace)::setDisplayNormalization(
 }
 
 /**
+  Return if the workspace is a histogram.
+*/
+TMDE(bool MDEventWorkspace)::isHistogramData() const {
+  return false;
```

Comment:
Why? Isn't an MD workspace always histogram data, in the sense of the `MatrixWorkspace::isHistogramData()` method?